domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -834,6 +883,34 @@ constructor(
             getResolutionSelector(sessionSettings.aspectRatio)
         )
 
+        if (currentSettings.value?.lowLightBoost == LowLightBoost.ENABLED) {
+            Log.d(TAG, "Adding Control AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY")
+            val camera2Interop = Camera2Interop.Extender(previewUseCaseBuilder)
+            if (Build.VERSION.SDK_INT >= 35) {
+                camera2Interop.setCaptureRequestOption(
+                    CaptureRequest.CONTROL_AE_MODE,
+                    CameraMetadata.CONTROL_AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY
+                )
```

Comment:
This should be done using `Camera2CameraControl` rather than adding it to the `Preview.Builder`. It's similar to the `CameraControl.setZoomRatio` or `setFlashModeInternal` that we set in the `cameraProvider.runWith {}` block.